plugins/modules/cloud/misc/proxmox.py

```diff
@@ -646,6 +668,7 @@ def main():
                             searchdomain=module.params['searchdomain'],
                             force=int(module.params['force']),
                             pubkey=module.params['pubkey'],
+                            features=",".join(module.params['features']),
```

Comment:
```suggestion
                            features=",".join(module.params['features'] or []),
```
To avoid an error if `features` is not specified.